sass/compat/themes/buddyboss-theme/__imports.scss

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------
+// DEFINE LOCAL VARIABLES
+// ------------------------------------
+$_body-page-selector:                  '.woocommerce-checkout' !default;
+$_body-theme-selector:                 '.theme-buddyboss-theme' !default;
+
+
+
+:root {
+	--fluidcheckout--field--height: #{ remsize( 4.4rem ) };
```

Comment:
Field height should be same as the height defined by the theme. 41px = 4.1rem for BuddyBoss